lib/voodoo_sms.rb

```diff
@@ -46,6 +46,12 @@ def get_sms(from, to, keyword = '')
     end
   end
 
+  def get_dlr_status(reference_id)
```

Comment:
I'm not sure whether it's best to keep this name reflecting the API method, or whether to use the full name `get_delivery_status`. When I first saw the API documentation, it took me a few moments to realise that dlr was shorthand for delivery